cmake/ProjectUVWASI.cmake

```diff
@@ -0,0 +1,53 @@
+if(ProjectUVWASIIncluded)
+    return()
+endif()
+set(ProjectUVWASIIncluded TRUE)
+
+include(ExternalProject)
+
+set(prefix ${CMAKE_BINARY_DIR}/_deps)
+set(source_dir ${prefix}/src/uvwasi)
+set(binary_dir ${prefix}/src/uvwasi-build)
+set(include_dir ${source_dir}/include)
+set(uvwasi_library ${binary_dir}/${CMAKE_STATIC_LIBRARY_PREFIX}uvwasi_a${CMAKE_STATIC_LIBRARY_SUFFIX})
+set(uv_library ${binary_dir}/_deps/libuv-build/${CMAKE_STATIC_LIBRARY_PREFIX}uv_a${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+if(UNIX AND NOT APPLE)
+    set(system_libs "pthread;dl;rt")
+endif()
+
+ExternalProject_Add(uvwasi
+    EXCLUDE_FROM_ALL 1
+    PREFIX ${prefix}
+    DOWNLOAD_NAME uvwasi-0.0.10.tar.gz
+    DOWNLOAD_DIR ${prefix}/downloads
+    SOURCE_DIR ${source_dir}
+    BINARY_DIR ${binary_dir}
+    URL https://github.com/cjihrig/uvwasi/archive/v0.0.10.tar.gz
```

Comment:
The version `0.0.11` has been released 20 days ago.

Reply:
I know, but it does not compile, so thought delaying it.